partcad/src/partcad/adhoc/convert.py

```
@@ -0,0 +1,91 @@
+        generate_partcad_config(temp_dir, input_type, input_path)
+
+        # Initialize PartCAD context and load the project
+        ctx = Context(root_path=temp_dir, search_root=False)
+        project = ctx.get_project("/")
+        part = project.get_part("input_part")
+        if not part:
+            raise RuntimeError("Failed to load the input part.")
+
+        # Export the part to the desired output format
+        export_part(part, output_filename, output_type, ctx)
+    except Exception:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)